public/styles/components/_forms.scss

```diff
@@ -227,6 +237,13 @@
       background: $brandColor;
     }
   }
+
+  &__add {
+    padding: 10px;
+    display: table;
```

Comment:
You wanna be 100% certain you want `display: table` here. It means a lot of CSS properties won't work on it any more

Reply:
@regiskuckaertz would know better than I the benefits / merits of this approach

Reply:
That's correct, the `padding` for instance can be removed here. The alternative (without changing the markup) would be to make the ancestor a flex column container and then flush-right the children, but I think this may have more side-effects.